blink/README.md

```diff
@@ -0,0 +1,40 @@
+# BlinkOps Integration
+
+## Overview
+
+The [Blink][1] integration is a collection of no-code actions and triggers usable in Blink Automations. Instead of writing manual API calls and scripts, you can use Blink's visual drag-and-drop editor to quickly automate your Datadog workspace, or use one of the hundreds of existing pre-made automations from the [library][2].
+
+This integration enables you to:
+1. Trigger automations within Blink in response to new Datadog incidents 
+2. Create and update Datadog incidents in Blink 
+3. View any created incidents or events in the Datadog event stream
```

Comment:
Do you mean any created incidents or events in Blink are viewable in Datadog?

Reply:
You can view created incidents or events in Blink, even if they weren't created by Blink themselves, since we use the API to get the event stream. I updated the readme to clarify this point